# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openttd/install-bin-location.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- openttd-15.1/CMakeLists.txt.vanilla	2026-01-26 14:17:31.102117508 +0000
+++ openttd-15.1/CMakeLists.txt	2026-01-26 14:21:50.278077982 +0000
@@ -27,7 +27,7 @@
     set(CMAKE_INSTALL_DATADIR "share/games")
 endif()
 if(NOT CMAKE_INSTALL_BINDIR)
-    set(CMAKE_INSTALL_BINDIR "games")
+    set(CMAKE_INSTALL_BINDIR "bin")
 endif()
 include(GNUInstallDirs)
 
